### ISSUE TITLE
ProductType list view query error

### ISSUE THREAD

**Issue by hetsch** [mirumee/saleor-dashboard#360]
### What I'm trying to achieve
…

### Steps to reproduce the problem
1. Using latest master branch of `saleor-dashboard` and `saleor`
2. In the dashboard open Configuration -> Product Types
3. ProductTypes list view is empty

### What I expected to happen
Seeing a list of all defined ProductTypes

### Possible solution
As far as I can tell, there is no `ProductTypeSortingInput` class in the Saleor backend anymore. At leas I could not find it in `saleor/graphql/product/schema.py`. What worked for me is:

```diff
diff --git a/src/productTypes/queries.ts b/src/productTypes/queries.ts
index f6bdc47..d78683f 100644
--- a/src/productTypes/queries.ts
+++ b/src/productTypes/queries.ts
@@ -53,7 +53,6 @@ export const productTypeListQuery = gql`
     $first: Int
     $last: Int
     $filter: ProductTypeFilterInput
-    $sort: ProductTypeSortingInput
   ) {
     productTypes(
       after: $after
@@ -61,7 +60,6 @@ export const productTypeListQuery = gql`
       first: $first
       last: $last
       filter: $filter
-      sortBy: $sort
     ) {
       edges {
         node {
```

**Comment by karolkielecki** [comment]
https://github.com/mirumee/saleor/blob/master/saleor/graphql/product/sorters.py#L214   Hi there! You can still find ProductTypeSortingInput in Saleor. We recommend you using Saleor 2.9 as it's the latest stable version. If you have more questions, please let me know.

**Comment by hetsch** [comment]
Thank you Karol. I have to check what went wrong on my side. Will try 2.9. Do you have any information on which version of `saleor`, `saleor-storefront`, and `saleor-dashboard` work with each other? Are the latest stable versions of each component compatible? Thank you!

**Comment by karolkielecki** [comment]
@hetsch Hey! My apologies for the late response. Yes, the latest stable versions should be compatible. If you have more questions, let us know. 
Best

**Comment by WahabShah23** [comment]
Hello, I was to open a new issue but wasn't sure if I should open new one or not. Then I stumbled upon this productType Listing. Actually I have kind of same error. And I am getting that for ProductTypes, Collections, and Categories. Their add edit (Mutations) work but for some reason Listing is not working (Query). I get the following error. The best part is the same queries are working fine on playground. What could be the issue?

**GraphQL error: 'AnonymousUser' object is not iterable**

![image](https://user-images.githubusercontent.com/14151483/87665619-a2d79480-c780-11ea-9a31-3f68e3e7e3f5.png)
